jmix-flowui/flowui-data/src/main/java/io/jmix/flowuidata/accesscontext/UiGenericFilterModifyGlobalConfigurationContext.java

```diff
@@ -18,11 +18,11 @@
 
 import io.jmix.core.accesscontext.SpecificOperationAccessContext;
 
-public class FlowuiGenericFilterModifyGlobalConfigurationContext extends SpecificOperationAccessContext {
+public class UiGenericFilterModifyGlobalConfigurationContext extends SpecificOperationAccessContext {
 
-    public static final String NAME = "flowui.genericfilter.modifyGlobalConfiguration";
+    public static final String NAME = "ui.genericfilter.modifyGlobalConfiguration";
```

Comment:
Should we rename it to `ui.filter.modifyGlobalConfiguration` in respect with `ui.filter.modifyJpqlCondition` and `ui.filter.modifyConfiguration`?